util/java/libtiled-java/src/main/java/org/mapeditor/io/TMXMapWriter.java

```diff
@@ -527,6 +535,10 @@ private void writeTile(Tile tile, XMLWriter w, String wp) throws IOException {
         w.startElement("tile");
         w.writeAttribute("id", tile.getId());
 
+		if (!tile.getType().isEmpty()) {
+			w.writeAttribute("type", tile.getType());
+		}
```

Comment:
Please make sure to indent with spaces rather than tabs.

Reply:
Whoops! fixed.